lldb/source/Plugins/Process/elf-core/RegisterContextPOSIXCore_riscv64.cpp

```diff
@@ -0,0 +1,84 @@
+//===-- RegisterContextPOSIXCore_riscv64.cpp ------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "RegisterContextPOSIXCore_riscv64.h"
+
+#include "lldb/Utility/DataBufferHeap.h"
+
+using namespace lldb_private;
```

Comment:
You have `lldb_private::` in a bunch of places in this file, but you have `using namespace lldb_private` here. Mind dropping the `lldb_private::` in this file for consistency?